pkg/util/util.go

```diff
@@ -284,3 +284,28 @@ func GetFileChecksum(filePath string) (string, error) {
 
 	return hex.EncodeToString(h.Sum(nil)), nil
 }
+
+func BackoffRetry(attempts int, baseDelay, maxDelay time.Duration, exponentialBackoff float64, backoffFunc func() error) error {
```

Comment:
Have you considered using `retry.RetryOnConflict` in k8s.io/client-go/util/retry instead? as go-common-libs depend on that client lib.